### PR TITLE
Update GH Actions and READMEs

### PR DESCRIPTION
**Pull request by ryanio**
This PR:
1. Normalizes workflow names
1. Runs workflows on PRs
1. Sets labeler to run on schedule so it can process forks (https://github.com/actions/labeler/issues/12#issuecomment-525762657)
1. Shortens workflow `node-version` to an array for brevity
1. Updates README badges
1. Adds a project root `.gitignore` for `node_modules` and `package-lock.json`

### CODE REVIEW

File: .github/workflows/account-test.yml [hunk 1]
```diff
@@ -1,6 +1,17 @@
-name: account-test
+name: Account Test
 'on':
   push:
+    branches:
+      - master
+    tags:
+      - '*'
+          paths-ignore:
```
Comment (evertonfraga) [inline .github/workflows/account-test.yml]: is this formatted correctly?

Comment (ryanio) [reply .github/workflows/account-test.yml]: nope, thanks for the catch

File: README.md [hunk 2]
```diff
@@ -49,3 +40,66 @@ If you want to join for work or do improvements on the libraries have a look at
...
+[tx-coverage-link]: https://codecov.io/gh/ethereumjs/ethereumjs-vm/tree/master/packages/tx
+[vm-package]: ./packages/vm
+[vm-npm-badge]: https://img.shields.io/npm/v/ethereumjs-vm.svg
+[vm-npm-link]: https://www.npmjs.org/package/ethereumjs-vm
+[vm-issues-badge]: https://img.shields.io/github/issues/ethereumjs/ethereumjs-vm/package:%20vm?label=issues
+[vm-issues-link]: https://github.com/ethereumjs/ethereumjs-vm/issues?q=is%3Aopen+is%3Aissue+label%3A"package%3A+vm"
+[vm-actions-badge]: https://github.com/ethereumjs/ethereumjs-vm/workflows/VM%20Test/badge.svg
+[vm-actions-link]: https://github.com/ethereumjs/ethereumjs-vm/actions?query=workflow%3A%22VM+Test%22
+[vm-coverage-badge]: https://codecov.io/gh/ethereumjs/ethereumjs-vm/branch/master/graph/badge.svg?flag=vm
+[vm-coverage-link]: https://codecov.io/gh/ethereumjs/ethereumjs-vm/tree/master/packages/vm
```
Comment (holgerd77) [inline README.md]: Uh, nice.

Comment (evertonfraga) [reply README.md]: That's much better :)

**Pull request merged**